react_redux/README.md

```diff
@@ -464,7 +442,7 @@
     }
     ```
 
-  - Bind event handlers for the render method in the constructor. eslint: [`react/jsx-no-bind`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md)
+  - Bind event handlers for the render method in the constructor. 
```

Comment:
_I think_ we can use `const onClickDiv = () => {...}` and `onClick={this.onClickDiv}` without `bind`. Would need to test